### PR TITLE
Fix SVDB/merge stubs

### PR DESCRIPTION
**Pull request by fellen31**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: modules/nf-core/svdb/merge/main.nf [hunk 1]
```diff
@@ -36,7 +36,10 @@ process SVDB_MERGE {
         $prio \\
         --vcf $input \\
         > ${prefix}.vcf
-    bgzip ${prefix}.vcf
+
+    bgzip \\
+        --threads ${task.cpus} \\
+        ${prefix}.vcf
```
Comment (Lucpen) [inline modules/nf-core/svdb/merge/main.nf:42]: Why not make a patch instead?

**Review by Lucpen — COMMENTED** [review]
Just one tiny thing

File: modules/nf-core/svdb/merge/main.nf (hunk 1 above)
Comment (fellen31) [reply modules/nf-core/svdb/merge/main.nf:42]: What do you mean? 😊

File: modules/nf-core/svdb/merge/main.nf [hunk 2]
```diff
@@ -48,7 +51,7 @@ process SVDB_MERGE {
     stub:
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
-    touch ${prefix}.vcf.gz
+    echo | gzip > ${prefix}.vcf.gz
```
Comment (Lucpen) [inline modules/nf-core/svdb/merge/main.nf:54]: why not just touch the file?

Comment (fellen31) [reply modules/nf-core/svdb/merge/main.nf:54]: 1.12.4 Stub gzip files must use echo and pipe in https://nf-co.re/docs/guidelines/components/modules

>Simply touching a file with the file name ending in .gz will break nf-test’s Gzip file parser, as the file is not actually >gzipped and thus cannot be read.
>
>Therefore we must make sure we generate a valid gzipped file for nf-test to accept it during tests.

**Review by Lucpen — APPROVED** [review]

**Pull request merged**